src/components/modals/Form.tsx

```diff
@@ -19,7 +19,9 @@
 import React, { useState } from 'react';
 import { Button, Form, Modal, Spinner } from 'react-bootstrap';
 import type { ModalProps } from 'react-bootstrap';
-import JsonSchemaForm from '@rjsf/bootstrap-4';
+// TODO: use @rjsf/react-bootstrap when @rjsf publishes v6 with support for
+// Bootstrap 5.
+import JsonSchemaForm from '@astarte-platform/react-bootstrap';
```

Comment:
i had a quick look at their repo and i didn't see an issue for the release of such version.
I would suggest creating one and linking it here so we can easily monitor their progress on the issue